virt/virt-4-12-release-notes.adoc

```diff
@@ -123,7 +123,7 @@ The SVVP Certification applies to:
 === Web console
 
 //CNV-20936 UI - live YAML view
-
+* When you customize a template to create a VM, you can set the *YAML* slider to *ON* on each VM configuration tab to view the live changes in the YAML configuration file alongside the form.
```

Comment:
I think normally if only "on/off" is provided, that element is called a switch, not a slider. See IBM SG: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#slider

Reply:
other suggestions:
- "on each VM configuration tab" - "in each VM configuration tab"
- alongside - next to

Reply:
IGS says the correct preposition for a tab is "on". https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tab